### ISSUE TITLE
Question: Is it possible to use this library in vscode extension? 

### ISSUE THREAD

**Issue by Manish3323** [mifi/build-electron#5]
Hi, 
This is not an issue/bug as such but a question from my end.
As an author of vscode extension. 
I am trying to dynamically load a config from a user's workspace. 

Currently electron's support is limited for usecase. 

Vscode being a electron app, do you think it is feasible for this library to fit in and patch this issue? 

I am trying to figure out a way of loading esm in a cjs package(vscode plugin). 
 Thanks

**Comment by mifi** [comment]
hi
not sure what you mean by this exactly, but this project is not made for vscode plugins, but for building electron apps, so I don't think that would be possible.

unrelated to build-electron, if you want to load esm from cjs, you can use `await import('./my-module.mjs')`

**Comment by Manish3323** [comment]
Hi thanks for the suggestion. 

I ve tried this. It's technically not feasible currently in vscode extension's node environment.

I just checked this PR https://github.com/electron/electron/pull/37535 which tries to solve the issue. As it tries to make electron app ESM compatible by default. Will wait for this pr to release. Hopefully future version of vscode will include this change and it will work.. 

So Closing this issue. Not a relevent place to open. My bad..
